### PR TITLE
Update hearings page content

### PR DESCRIPTION
**Pull request by cehsu**
These changes update the hearings page content.

localhost_3001_disability-benefits_claims-appeal_hearings_ 3

### CODE REVIEW

**Review by gnakm — APPROVED** [review]

File: content/pages/disability-benefits/claims-appeal/hearings.md [hunk 1]
```diff
@@ -6,63 +6,126 @@ plainlanguage: 11-4-16 certified in compliance with the Plain Writing Act
...
+
+### How will I know when my hearing is scheduled?
+
+You’ll receive a notice in the mail at least 30 days before your hearing is scheduled.
+
+### What if I need to reschedule my hearing?
+
+Send a written request to reschedule your hearing at least 2 weeks before your scheduled hearing. Include your name and the VA file number for your claim.
+
+**Mail your request to:**
```
Comment (cvalarida) [inline content/pages/disability-benefits/claims-appeal/hearings.md]: I think the convention is to make this a heading (h4 or h5). @bethpotts to confirm.

File: content/pages/disability-benefits/claims-appeal/hearings.md [hunk 2]
```diff
@@ -6,63 +6,126 @@ plainlanguage: 11-4-16 certified in compliance with the Plain Writing Act
...
-  Director, Office of Management, Planning and Analysis (014)<br/>
-  Board of Veterans’ Appeals<br/>
-  PO Box 27063<br/>
-  Washington, DC 20038<br/>
+Board of Veterans’ Appeals<br>
+PO Box 27063<br>
+Washington, DC 20038<br>
 </p>
+
+<span id="preparing"></span>
```
Comment (cvalarida) [inline content/pages/disability-benefits/claims-appeal/hearings.md]: There are a few of these blank tags. What are they for?

**Review by cvalarida — APPROVED** [review]
My main questions surround what should be in a header and what should be just bold. I'll leave that to @gnakm and @bethpotts, though.

Also, we should write up a ticket to add schema tags to this content for SEO: http://schema.org/HowTo
More broadly, we should probably search for _all_ static content that doesn't have these tags. :thinking:

File: content/pages/disability-benefits/claims-appeal/hearings.md (hunk 2 above)
Comment (cehsu) [reply content/pages/disability-benefits/claims-appeal/hearings.md]: These are skip links. We have set them up this way throughout, however, we should probably use [sections](https://webaim.org/techniques/skipnav/) instead. Ticket created [here](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/9186).

**Comment by cehsu** [comment]
PR comments addressed. @bethpotts plans to take a look at these changes today/tomorrow.

File: content/pages/disability-benefits/claims-appeal/hearings.md (hunk 2 above)
Comment (cvalarida) [reply content/pages/disability-benefits/claims-appeal/hearings.md]: Ahhh, skip links. Makes sense. Thanks! :+1:

**Comment by bethpotts** [comment]
@cehsu or @cvalarida Can you add a little more space at the bottom of the page before the footer? And the Form 9 pdf didn't load for me. Is there another link we could use? Or is it just slow because we're in preview here? Thanks!

**Review by bethpotts — APPROVED** [review]
Couple questions for Claire/Chris above, but otherwise, the content looks good.

**Comment by gnakm** [comment]
@cehsu @cvalarida I added changes to Beth's branch after chatting with her. After those changes, everything is good to go! 👍

**Comment by cehsu** [comment]
Thanks @bethpotts @gnakm , I've updated the screenshot in the description. Does the spacing above the footer look alright? Also, the link to VA Form 9 works, though it may take a second.

**Pull request merged**